templates/zerver/meta_tags.html

```diff
@@ -12,6 +12,9 @@
 {% if OPEN_GRAPH_TITLE %}
 <meta property="og:title" content="{{ OPEN_GRAPH_TITLE }}">
 <meta property="og:description" content="{{ OPEN_GRAPH_DESCRIPTION }}">
+{% elif realm_name and not realm_invite_required and not realm_emails_restricted_to_domains %}
+<meta property="og:title" content="{{ realm_name }}">
+<meta property="og:description" content="Participate in {{ realm_name }} by joining their Zulip. Zulip is the world's most productive team chat.">
```

Comment:
It's possible we should put this code in `zerver/context_processors.py` and set `OPEN_GRAPH_TITLE` and `OPEN_GRAPH_DESCRIPTION` to avoid doing a complex conditionals in HTML?